.env

```diff
@@ -0,0 +1,22 @@
+# MongoDB
+MONGO_PORT=27017
+MONGO_INITDB_ROOT_USERNAME=admin
+MONGO_INITDB_ROOT_PASSWORD=password
+
+## Hardened MongoDB
+MONGO_INITDB_DATABASE=nlpsandbox
```

Comment:
should these be changed from `nlp` to `rocc`?

Reply:
Yes

Reply:
Use `MONGO_INITDB_DATABASE=rocc`

Reply:
Ah okay, wasn't sure if I could change these values.